### PR TITLE
Add to the readme

### PR DESCRIPTION
**Pull request by Cooperbuilt**
## Description
Adding to the readme

Works towards the progress of #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,26 @@
 # Awesome Learning
 [![Build Status](https://travis-ci.org/wayfair/awesome-learning.svg?branch=master)](https://travis-ci.org/wayfair/awesome-learning)
 
-Learn JavaScript and Front-End Fundamentals at your own pace.
+Awesome Learning is a Frontend focused learning platform created by current and former members of Wayfair Engineering.
```
Comment (SheaBelsky) [inline README.md]: `front-end` or `Front-end`

File: README.md [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
-[https://wayfair.github.io/awesome-learning/](https://wayfair.github.io/awesome-learning/)
+[Check out the site](https://wayfair.github.io/awesome-learning/)
 
 ## Quick Start
 
-#### Start Developing
+#### Working With Our Repo
+1. Fork the Awesome Learning Repo
+2. Clone your fork to your local directory
+3. Change into the local directory
```
Comment (SheaBelsky) [inline README.md]: `Change` => `Navigate` (Clearer intent)

File: README.md [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
+4. Install dependencies
+```sh
+npm install
+```
+
+#### Starting the Repo
```
Comment (SheaBelsky) [inline README.md]: I'd recommend changing this to `Starting Awesome Learning`. I don't know how much sense `Starting the Repo` makes; it's not just a repo, it's the Awesome Learning platform.

File: README.md [hunk 4]
```diff
@@ -20,56 +29,50 @@ Your site is now running at `http://localhost:8000`!
...
+#### Running Tests
 
+While you are developing, we recommend watching the test suite. Open up another terminal in the site directory and run 
+```
+npm run test-watch
+```
 
-## Folder Structure
+## Project Structure
+Awesome Learning is powered by two repositories, this one and our [exercises repo](https://wayfair.github.io/awesome-learning-exercises/).
```
Comment (SheaBelsky) [inline README.md]: I think you mean to link to `https://github.com/wayfair/awesome-learning-exercises` here.

File: README.md [hunk 5]
```diff
@@ -20,56 +29,50 @@ Your site is now running at `http://localhost:8000`!
...
-    │       └── Menu
-    ├── constants
-    ├── templates
-    └── utils
+#### This Repo
+This is a Gatsby.js powered static site. We use GraphQL as our data layer (see [Gatsby GraphQL](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql)), React for our components, co-located SCSS for our styling, and Jest/React Testing Library for our automated tests.
 
-```
+#### The Exercises Repo
+The exercises repo holds the courses and lessons that power the exercise portions of each lesson. You do not need to pull down the exercises repo in order to contribute to Awesome Learning. The exercises repo acts as a target for a codesandbox API which pulls down the files and creates an exercise codesandbox on the fly. For more information on how that works, check out [importing from git](https://codesandbox.io/docs/importing#import-from-github) and the exercises repo readme.
```
Comment (SheaBelsky) [inline README.md]: Capitalize `CodeSandbox`

File: README.md [hunk 6]
```diff
@@ -20,56 +29,50 @@ Your site is now running at `http://localhost:8000`!
...
 
-## Credits
-Credit to the original author of this fork - the Gastby Lumen Starter.
-https://github.com/alxshelepenok/gatsby-starter-lumen
+If you are interested in fixing exercise content, and/or contributing content, please checkout the [exercises repo](https://wayfair.github.io/awesome-learning-exercises/) and we help you out from there.
```
Comment (SheaBelsky) [inline README.md]: You should correct the link to the GitHub repo here.

File: README.md [hunk 7]
```diff
@@ -20,56 +29,50 @@ Your site is now running at `http://localhost:8000`!
...
+
+
+# What is Awesome Learning?
+ This platform features a series of courses designed to help you thoroughly learn technical topics. Some of our courses include Frontend Testing, Javascript Data Types, and Javascript Array Methods. Each course contains multiple lessons designed to teach one aspect of the course topic. A lesson has pre-read materials, learning materials, pre-read quizzes, and hands-on exercises.
```
Comment (SheaBelsky) [inline README.md]: `Javascript` => `JavaScript`

Comment (Cooperbuilt) [reply README.md]: done

**Comment by Cooperbuilt** [comment]
All comments resolved, ready for another look @SheaBelsky

**Pull request merged**